libs/common-services/src/main/java/uk/gov/di/ipv/core/library/gpg45/Gpg45Scores.java

```diff
@@ -111,7 +111,9 @@ public List<Gpg45Scores> calculateGpg45ScoresRequiredToMeetAProfile(
                             }
 
                             return new Gpg45Scores(
-                                    missingEvidence,
+                                    missingEvidence.isEmpty()
```

Comment:
If we change the `evidence.size() <= 1` above, I don't think we should need this - and I think it's more accurate to say no evidence is required(?)